examples/sdk/node/src/index.ts

```diff
@@ -0,0 +1,67 @@
+import { BacktraceClient } from '@backtrace/node';
+import fs from 'fs';
+import path from 'path';
+import { exit } from 'process';
+import readline from 'readline';
+import { SUBMISSION_URL } from './consts';
+const reader = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+const client = BacktraceClient.builder({
+    url: SUBMISSION_URL,
+    attachments: [path.join(path.dirname(process.cwd()), 'samplefile.txt')],
+    rateLimit: 5,
+    userAttributes: {
+        'custom-attribute': 'test',
+        'complex-attribute': {
```

Comment:
This will be used as an annotation, right? Maybe we should not confuse the user and name this `custom-annotation`?

Reply:
changed. Thx